### DISCUSSION TITLE
Welcome to AetherOnePi Discussions!

### DISCUSSION THREAD

**Discussion by isuretpolos** [isuretpolos/AetherOnePi#31] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cstewart28** [comment]
Currently working on getting to work on RasberryPi 4 and touchscreen and camera.  Would like if you had considered making the screen resolution dynamic
